Libraries/Image/__tests__/resolveAssetSource-test.js

```diff
@@ -159,6 +160,34 @@ describe('resolveAssetSource', () => {
       });
     });
   });
+  
+  describe('bundle was loaded from file on Android', () => {
+    beforeEach(() => {
+      NativeModules.SourceCode.scriptURL =
+        '/sdcard/Path/To/Simulator/main.bundle';
```

Comment:
@frantic, I guess the confusion is that I thought this was expected behaviour. In [this line](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java#L187), the prescribed usage is to pass in the path with the "assets://" prefix for asset bundles, but NO prefix for file bundles. What do you think should be the best way forward? Should we mandate users to pass in file bundle paths with the "file://" prefix, or keep the current usage, and return the source URL prefixed with "file://" [here](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/JSBundleLoader.java#L40)? Like for example, 
```java
return fileName.startsWith("assets://") ? fileName : "file://" + fileName;
```